packages/react-button/src/components/CompoundButton/CompoundButton.scss

```diff
@@ -0,0 +1,56 @@
+.root {
```

Comment:
can remove this file now

Reply:
Talked with David about this, he doesn't want to remove the scss files yet until we get a better sense of the performance differences.

Reply:
Yeah, let's add it, keep it, and validate updates in ribbon. We can remove it later this week assuming all is well.